scripts/qa_crowbarsetup.sh

```diff
@@ -2141,6 +2141,13 @@ function onadmin_allocate()
         done
     fi
 
+    # set rootfs for all nodes when want_rootfs is set
+    if [ -n "$want_rootfs" ] ; then
+        for node in `get_all_discovered_nodes` ; do
+            set_node_fs $node "$want_rootfs"
```

Comment:
with this code you can do want_docker=1 want_rootfs=xfs and it would not deploy docker with btrfs but with xfs.